guardian/amigo
# removes the s3-ssh-keys role

## What does this change?


We no longer support direct SSH onto instances and favour SSH over SSM. To enforce this, remove the role that made direct SSH access easy; essentially make it hard to do the wrong thing.

This requires all recipes to stop using this role. There is one remaining and it is being migrated (https://github.com/guardian/domains-platform/pull/54 and https://github.com/guardian/domains-platform/pull/55).

## How to test


- Deploy to CODE
- Open recipe
- Witness s3-ssh-keys role is no longer available
- Run a bake
- Witness AMIs continue to bake

## How can we measure success?


Ultimately, the enforcement of security best practice.

## Have we considered potential risks?


Recipes that require this role will stop working. There is only one in PROD and it is being migrated; dependant PRs are linked above.

## Images


n/a
